src/ngraph/runtime/cpu/op/batch_mat_mul_transpose.cpp

```diff
@@ -25,11 +25,13 @@
 using namespace std;
 using namespace ngraph;
 
-op::BatchMatMulTranspose::BatchMatMulTranspose(const shared_ptr<Node>& arg0,
-                                               const shared_ptr<Node>& arg1,
+const std::string op::BatchMatMulTranspose::type_name{"BatchMatMulTranspose"};
+
+op::BatchMatMulTranspose::BatchMatMulTranspose(const Output<Node>& arg0,
```

Comment:
One minor thing, all tests now rely on implicit conversion between `node` and `Output<Node>`. Maybe we should add a test to use `Output<Node>` directly when constructing nodes, and perhaps use non-zero index ?

Reply:
Once all node class constructors have been upgraded, we can tighten things up some.